### ISSUE TITLE
Bug in class gwnet(nn.Module)

### ISSUE THREAD

**Issue by lrk612** [nnzhan/Graph-WaveNet#41]
`len(supports)` in `self.gconv.append(gcn(dilation_channels,residual_channels,dropout,support_len=len(supports)))` should be replaced as `self.supports_len`.
Otherwise when no supports are passed in, since self.supports is an empty list, it will cause the dimension c_in of the mlp in gcn to not match the data. I think this is why the author specifically maintained the variable self.supports_len earlier and +1
